test/base64_unittest.cc

```diff
@@ -0,0 +1,62 @@
+#include "../src/base64.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(EncodeTest, EmptyEncode) {
+  EXPECT_EQ("", base64::Encode(""));
+}
+
+TEST(EncodeTest, SimpleEncode) {
+  EXPECT_EQ("dGVz", base64::Encode("tes"));
+}
+
+// Base64 encoders typically pad messages to ensure output length % 4 == 0. To
+// acheive this, encoders will pad messages with either one or two "=".  Our
```

Comment:
Let's stick to one space after "." (throughout).

Reply:
Fixed.